### PR TITLE
Fix Spelling/English mistakes

### PR DESCRIPTION
**Pull request by danzel**
Discovered by @Dragios in #2878

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/2879)

### CODE REVIEW

**Comment by FearlessTobi** [comment]
LGTM

File: src/input_common/main.h [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace InputCommon {
 /// Initializes and registers all built-in input device factories.
 void Init();
 
-/// Unresisters all build-in input device factories and shut them down.
+/// Deregisters all built-in input device factories and shuts them down.
```
Comment (Subv) [inline src/input_common/main.h]: I myself prefer "unregisters" but 🤷‍♂️

**Comment by jroweboy** [comment]
Erm. Did anyone actually read the english stack exchange link? Its pretty clear that unregister has taken over in common usage, but i literally don't care...

**Comment by danzel** [comment]
hahaha 🤦‍♂️ , yeah looks a bit subjective. I think either is fine, it was spelt incorrectly before anyway.

**Pull request merged**